pkg/monitoring/metrics/virt-controller/vmstats_collector.go

```diff
@@ -122,7 +122,7 @@ var (
 			Name: "kubevirt_vm_resource_requests",
 			Help: "Resources requested by Virtual Machine. Reports memory and CPU requests.",
```

Comment:
can we update the description with the new label? 
and a question for me - the source can be domain/gust/hugepages? is there more options?

Reply:
I don't think so